dotnetv3/Keyspaces/Scenarios/KeyspacesBasics.cs

```diff
@@ -0,0 +1,230 @@
+            var resp = new GetTableResponse();
+            Console.WriteLine("Waiting for the new table to be active. ");
+            do
+            {
+                try
+                {
+                    resp = await keyspacesWrapper.GetTable(keyspaceName, tableName);
+                    Console.Write(". ");
+                }
+                catch (ResourceNotFoundException ex)
+                {
+                    Console.WriteLine("... ");
+                }
+            } while (resp.Status != TableStatus.ACTIVE);
+
+            // Display the table's schema.
+            Console.WriteLine($"\nTable {tableName} has been created in {keyspaceName}");
+            Console.WriteLine("Let's take a look at the schema.");
+            uiMethods.DisplayTitle("All columns");
+            resp.SchemaDefinition.AllColumns.ForEach(column =>
+            {
+                Console.WriteLine(column.Name);
+            });
+            uiMethods.DisplayTitle("Cluster keys");
+            resp.SchemaDefinition.ClusteringKeys.ForEach(clusterKey =>
+            {
+                Console.WriteLine(clusterKey.Name);
+            });
+            uiMethods.DisplayTitle("Partition keys");
+            resp.SchemaDefinition.PartitionKeys.ForEach(partitionKey =>
+            {
+                Console.WriteLine(partitionKey.Name);
+            });
+
+            uiMethods.PressEnter();
+        }
+        catch (ResourceNotFoundException ex)
+        {
+            Console.WriteLine($"Error: {ex.Message}");
+        }
+
+        // Access Apache Cassandra using the Cassandra drive for C#.
+        var cassandraWrapper = new CassandraWrapper();
+        var movieFilePath = configuration["MovieFile"];
+
+        Console.WriteLine("Let's add some movies to the table we created.");
+        var inserted = await cassandraWrapper.InsertIntoMovieTable(keyspaceName, tableName, movieFilePath);
+
+        uiMethods.PressEnter();
+
+        Console.WriteLine("Added the following movies to the table:");
+        var rows = await cassandraWrapper.GetMovies(keyspaceName, tableName);
+        foreach (var row in rows)
+        {
+            var title = row.GetValue<string>("title");
+            var year = row.GetValue<int>("year");
+            var plot = row.GetValue<string>("plot");
+            Console.WriteLine($"{title}\t{year}\n{plot}");
+            Console.WriteLine(uiMethods.SepBar);
+        }
+
+        // Update the table schema
+        uiMethods.DisplayTitle("Update table schema");
+        Console.WriteLine("Now we will update the table to add a boolean field called watched.");
+
+        // First save the current time as a UTC Date so the original
+        // table can be restored later.
+        var timeChanged = DateTime.UtcNow;
+
+        // Now update the schema.
+        var resourceArn = await keyspacesWrapper.UpdateTable(keyspaceName, tableName);
+        uiMethods.PressEnter();
+
+        // TODO: Change this to update multiple movies.
```

Comment:
There is a TODO left in here.

Reply:
It was to remind me to write code to update more than one movie. I've done that now and the TODO comment is gone.